tests/testthat/test-4IO.R

```diff
@@ -493,8 +495,8 @@ test_that("Import HUMAnN file", {
     file_path <- system.file("extdata", "humann_output.tsv", package = "mia")
     tse <- importHUMAnN(file_path)
     #
-    expect_true( length(taxonomyRanks(tse)) == 7 )
-    expect_true( ncol(rowData(tse)) == 9 )
+    expect_true( length(taxonomyRanks(tse)) == 2 )
+    expect_true( ncol(rowData(tse)) == 4 )
```

Comment:
Now only levels that have information is returned. Previously the function returned always the whole taxonomy (Kingdom, Phylum,... Species). Now only Genus and Species are returned since those are only ranks found from data.